backends/redis.js

```diff
@@ -5,9 +5,13 @@ module.exports = function(Model, options) {
   if (options.ttl != null && options.ttl) {
     // Assuming the Model connector is "redis" here but we may need a config in
     // the future.
+
     Model.observe('after save', function(ctx, next) {
       // @see http://redis.io/commands/expire.
-      // TODO
+      var redisClient = ctx.Model.getConnector().client;
+      //pass arguments as array
+      redisClient.expire([ctx.Model.modelName+":"+ctx.instance.id, options.ttl]);
```

Comment:
It'd expect a callback, you could pass the `next` to it but let me know if you got different opinions.